app/templates/components/navigation/nav-desktop.html

```diff
@@ -0,0 +1,13 @@
+<ul
+    id="accordion-list"
+    class="h-full px-mobile py-7 overflow-scroll max-w-hero"
+>
+    <li class="border-b border-corporate-blue-light text-bold mb-6">
+        <h3 class="font-bold">{{ translations.nav.subHeader_1 }}</h3>
+    </li>
```

Comment:
I would move this header outside of the list, since it’s rather labeling/describing the list.